### PR TITLE
Healtcare tech doc

### PR DESCRIPTION
**Pull request by mdewey**
Added some answer and questions

### CODE REVIEW

File: products/health-care/questionnaire/discovery/engineering/README.md [hunk 1]
```diff
@@ -111,10 +124,47 @@ My desired next steps if I were to continue down this route:
 
 Currently, the only way the update the form content, both the structure and the text the user sees, is through a pull request to the code base. This is a technical process, that is built for developers. This might be a very high level to get this to work properly.
 
+> What is session storage and what are its limits?
+
+Session storage is a client side, temporary data store that is perfect for short term storage. The data lives client side in the browser window until the window is closed. This allows a user's old answers to be stored when the user navigates away and saved when the user navigates back.
+
+This idea breaks down when if the vet leaves the site and and comes back, all unauthenticated. If we care to save the data, we could build aa system that allows saving of non-authenticated users data. We can do this by saving an instance of the form filling out with a unique id and saving the answers related to that unique id, until and if the user logs in. There are security and other considerations that need to be flushed out.
```
Comment (abuckley3385) [inline products/health-care/questionnaire/discovery/engineering/README.md:131]: this looks good

File: products/health-care/questionnaire/discovery/engineering/README.md [hunk 2]
```diff
@@ -314,8 +372,8 @@ After the initial demo, we have found a list questions that need to be answered.
 
 ## VA Gov research using vets-web frontend
 
-The vets-web front end mocks some of the data, that is it has 
+The vets-web front end mocks some of the data, that is it has
```
Comment (abuckley3385) [inline products/health-care/questionnaire/discovery/engineering/README.md:375]: looks good

**Review by abuckley3385 — CHANGES_REQUESTED** [review]
please see my couple questions

File: products/health-care/questionnaire/discovery/engineering/README.md (hunk 1 above)
Comment (mdewey) [reply products/health-care/questionnaire/discovery/engineering/README.md:131]: Thank you

File: products/health-care/questionnaire/discovery/engineering/README.md (hunk 2 above)
Comment (mdewey) [reply products/health-care/questionnaire/discovery/engineering/README.md:375]: Thank you

**Comment by abuckley3385** [comment]
confirm

**Pull request merged**